python/pyarrow/dataset.py

```diff
@@ -798,6 +800,20 @@ def write_dataset(data, base_dir, basename_template=None, format=None,
         used determined by the number of available CPU cores.
     max_partitions : int, default 1024
         Maximum number of partitions any batch may be written into.
+    max_open_files : int, default 1024
+        Maximum number of number of files can be opened
+    max_rows_per_file : int, default 0
+        Maximum number of rows per file
+    min_rows_per_group : int, default 0
+        Minimum number of rows per group. When the value is greater than 0,
+        the dataset writer will batch incoming data and only write the row
+        groups to the disk when sufficient rows have accumulated.
+    max_rows_per_group : int, default 1 << 20
```

Comment:
Could we instead write the default like `1024 * 1024`? I find that easier to think about and I don't think I'm alone in that.

```suggestion
    max_rows_per_group : int, default 1024 * 1024
```

Reply:
I agree, it is more readable.